### DISCUSSION TITLE
the SD and SEM of the gmx_MMPBSA

### DISCUSSION THREAD

**Discussion by sf-polaris** [Valdes-Tresanco-MS/gmx_MMPBSA#440] [Q&A]
![image](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/2091c36c-f38a-4c8e-ba38-f90e4a949aa7)
Hi, I  run the MD and use 20 frames to calculate  the energy decomposition of each residue,  in the "decomposition dat" files, I find the SD and SEM are large, but when I calculate the SD using the " decomposition csv" files  by myself ， the SD is small. How does the "decomposition dat" file calculate the SD ?

**Comment by marioernestovaldes** [comment]
please send the files you are using to see if something is wrong

**Reply by sf-polaris** [reply]
[FINAL_DECOMP_MMPBSA.csv](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/13626428/FINAL_DECOMP_MMPBSA.csv)
[FINAL_DECOMP_MMPBSA2.xlsx](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/13626429/FINAL_DECOMP_MMPBSA2.xlsx)

Here are my two output files.
the ".csv" files don't calculate the SD and SEM ,  I converted the ".dat" files to the ".xlsx" files because GitHub cannot send the ".dat" files, and the ".dat" files calculate the SD and SEM. The SD in the  ".dat" files  are large. I want to know how does the gmxMMPBSA calculate the SD?

**Comment by marioernestovaldes** [comment]
The SDs are calculated using propagation of uncertainty rules (https://en.wikipedia.org/wiki/Propagation_of_uncertainty) and SD using arrays of values. We may revisit this at some point in the future though...
